Website/pages/docs/index.mdx

```diff
@@ -24,3 +47,15 @@ Drifty is an Open-Source project that welcomes contributions from the community.
 - **[Join our Discord Community](https://discord.gg/DeT4jXPfkG)**: It is the best place to get help, share your ideas, and connect with maintainers and other users/developers.
 - **[Join our GitHub Discussions](https://github.com/SaptarshiSarkar12/Drifty/discussions)**: Participate in discussions, ask questions, and share your thoughts on Drifty. We would love to hear from you!
 - **[Learn how to contribute](Contributing.md)**: If you are interested in contributing to Drifty, check out our guidelines and get started with your first contribution.
+
+## Look and Feel
+
+A simple demo on how to get started with the GUI of Drifty
+
+<Cards>
+  <Card title="Download" href="/download" />
+  <Card
+    title="Demo"
+    href="https://github.com/SaptarshiSarkar12/Drifty/assets/105960032/5bb21bf6-fb2b-4374-8d0b-14cac3c06908"
+  />
+</Cards>
```

Comment:
I believe it would be more efficient to only load the video when the user chooses to watch it. By rendering it by default, we risk increasing data consumption and slowing page performance, especially for individuals with poorer internet speeds.

Furthermore, we anticipate that the demo videos will be of particular interest to new users. Regular users are unlikely to want to watch it every time they visit the page, which could impact their overall experience.

As discussed, I will try to implement the videos to load in a popup on demand.

I'd welcome your feedback on this method, as it was just an idea I got while working on it.

Reply:
@abhilashreddysh I appreciate your thought. Yes, you can try out how you can load videos on demand.